examples/demo/site/modules/auth/mail_templates/account_activation.html

```diff
@@ -8,11 +8,7 @@
 	</head>
 
 	<body>
-		<p>Thank you for registering at <a href="$host">ROC CMS</a></p>
-
-		<p>To complete your registration, please click on this link to activate your account:<p>
-
-		<p><a href="$link">$link</a></p>
-		<p>Thank you for joining us.</p>
+		<p>Thank you for applying to  <a href="$host">ROC CMS</a> $user</p>
```

Comment:
For here and all the other templates, ... replace ROC CMS .. by $sitename ... and provide the site's name instead of ROC CMS

in our case, it will be "eiffel.org" or even "the Eiffel community site"